src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -24,6 +24,6 @@ public void end(boolean interrupted) {
     }
 
     public boolean isFinished() {
-        return intakeSystem.isNoteInside();
+        return false;
```

Comment:
why did you remove the condition?

Reply:
for making possible to use the IntakeCommand and move the intake wheels with a note in intake